### PR TITLE
deprecate cuda 11.5 environment

### PR DESCRIPTION
**Pull request by savitamittal1**

### CODE REVIEW

File: training/general/src/environments/acpt-pytorch-1.11-cuda11.5/spec.yaml [hunk 1]
```diff
@@ -14,6 +13,8 @@ build:
 os_type: linux
 
 tags:
+  Deprecated: ""
+  DeleteOn: "2023-04-14"
```
Comment (vizhur) [inline training/general/src/environments/acpt-pytorch-1.11-cuda11.5/spec.yaml:17]: We are not deleting it. We do not support metadata update yet (i.e. archive)

Comment (savitamittal1) [reply training/general/src/environments/acpt-pytorch-1.11-cuda11.5/spec.yaml:17]: ok, so just keep the deprecate tag?

Comment (savitamittal1) [reply training/general/src/environments/acpt-pytorch-1.11-cuda11.5/spec.yaml:17]: Also how do we make sure, its removed from vulnerabilities?

**Review by elesel — APPROVED** [review]

File: training/general/src/environments/acpt-pytorch-1.11-cuda11.5/spec.yaml (hunk 1 above)
Comment (savitamittal1) [reply training/general/src/environments/acpt-pytorch-1.11-cuda11.5/spec.yaml]: thanks @elesel . I have removed the tag 'DeleteOn'

**Review by cassieesvelt — APPROVED** [review]

**Pull request merged**